multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/helpers/MtaDescriptorPropertiesResolver.java

```diff
@@ -53,6 +57,9 @@ public DeploymentDescriptor resolve(DeploymentDescriptor descriptor) {
         descriptor = correctEntityNames(descriptor);
         // Resolve placeholders in parameters:
         CloudHandlerFactory handlerFactory = context.getHandlerFactory();
+        // add live route as reference to its route because if we don't do it, the resolve will fail, if the user wants to use the
+        // live-route
+        replaceNullLiveRoutesWithReferenceToRoute(descriptor);
```

Comment:
make a new resolver class, that will resolve and return the new descriptor with the replaced null values